README.md

```diff
@@ -1,2 +1,47 @@
-# lab-09
-API Server
+# LAB - 09
+
+## Api Server
+
+### Author: Meron 
+
+### Links and Resources
+* [submission PR](https://github.com/meron-401n14/lab-09/pull/1)
+* [travis](http://xyz.com)
+* [Heruku](http://xyz.com) 
+
+
+#### Documentation
+* [api docs](http://xyz.com) (API servers)
+* [jsdoc](http://xyz.com) (Server assignments)
+* [styleguide](http://xyz.com) (React assignments)
+
+### Modules
+#### `modulename.js`
+##### Exported Values and Methods
+
+###### `foo(thing) -> string`
+Usage Notes or examples
+
+###### `bar(array) -> array`
+Usage Notes or examples
+
+### Setup
+#### `.env` requirements
+* `PORT` - 3000
+* `MONGODB_URI` - "mongodb+srv://meron123:meron123@cfcluster-kexaa.mongodb.net/app?retryWrites=true&w=majority"
+
+#### Running the app
+* `npm start`
+* Endpoint: `/model/models/:model/schema/`
```

Comment:
This doesn't seem like a valid endpoint, I believe valid endpoints are `/models` or `/:model/schema`